recipes/poap/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "POAP" %}
+{% set version = "0.1.26" %}
+{% set sha256 = "05d34b8075faf4bc41e1a169151f545ef3e45cfa62ec31f815fa3c06ee2dd4fb" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
Could this be a `noarch` package?

Reply:
Hmm, I think so, there are no binaries and no visible dependency on platform. Since it passed on all platforms, I'll give a try.